app/views/idv/document_capture/show.html.erb

```diff
@@ -13,4 +13,4 @@
       skip_doc_auth_from_handoff: skip_doc_auth_from_handoff,
       doc_auth_selfie_capture: doc_auth_selfie_capture,
       mock_client: mock_client,
-    ) %>
+    ) %>
```

Comment:
Can you add the additional line back in?  No need to have this file show up in the diff list